code_of_conduct.md

```diff
@@ -0,0 +1,133 @@
+
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
```

Comment:
i don't know if it makes a difference, but in this [doc](https://github.com/github/docs/blob/main/content/communities/setting-up-your-project-for-healthy-contributions/adding-a-code-of-conduct-to-your-project.md), the file name appears in uppercase.